cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/utils/NGSICharsets.java

```diff
@@ -41,9 +41,9 @@ public static String encodePostgreSQL(String in) {
             char c = in.charAt(i);
```

Comment:
Where encodePostgreSQL() function is used?

It seems this PR fixes it but it is not clear to me where it is used...

Reply:
When "Enable ecoding=true", encodePostgreSQL() function is used while building schema name, db name and table name.